src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -229,7 +235,10 @@ function validatePropTypes(element) {
 
 var ReactElementValidator = {
   createElement: function(type, props, children) {
-    var validType = typeof type === 'string' || typeof type === 'function';
+    var validType =
+      typeof type === 'string' ||
+      typeof type === 'function' ||
+      type === REACT_FRAGMENT_TYPE;
```

Comment:
This should probably just allow any symbol and numbers so that we can use this space for other built-in fun things. E.g. passing it to host configs.